program_repair.py

```diff
@@ -22,19 +26,42 @@ def replace(input_string, line, column, length, new_string):
     return result
 
 
+def make_mutants(input_string, line):
+    mutants = []
+    length = len(input_string.splitlines()[line - 1]) + 2
```

Comment:
This feels like a lot of magic, why is the + 2 needed? Can you explain in a comment?